package.json

```diff
@@ -7,18 +7,17 @@
         "classnames": "2.3.1",
         "enzyme": "3.11.0",
         "enzyme-adapter-react-16": "1.15.6",
-        "node-sass": "4.14.1",
+        "node-sass": "7.0.0",
```

Comment:
I didn't see that you had updated this already when I commented earlier and filed [C2LC-578](https://issues.fluidproject.org/browse/C2LC-578).  If we're already updating, then we should go ahead and update to `7.0.1`.

Reply:
Done. Thanks.